lib/google_calendar.rb

```diff
@@ -1,33 +1,37 @@
+require 'googleauth'
+require 'google/apis/calendar_v3'
+
 class GoogleCalendar
   def initialize(config, logger)
     @config = config
 
     if @config['google']['key'].present?
-      @key = OpenSSL::PKCS12.new(@config['google']['key'], @config['google']['key_secret']).key
+      # https://github.com/google/google-auth-library-ruby/issues/65
+      # https://github.com/google/google-api-ruby-client/issues/370
+      ENV['GOOGLE_PRIVATE_KEY'] = @config['google']['key']
```

Comment:
Is it possible to authenticate without using the environment? As far as I know the `ENV` persists for the current process and 'leaks' to all other Agents.